File: java/javadpbo/src/com/sem4dpbo/week2/contohkasus/HasilKerajinan.java
package com.sem4dpbo.week2.contohkasus;

public class HasilKerajinan {
    private int kodeKerajinan;
    private String bahan, namaKerajinan;

    HasilKerajinan(){}

    public void setKodeKerajinan(int kodeKerajinan) {
        this.kodeKerajinan = kodeKerajinan;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public void setNamaKerajinan(String namaKerajinan) {
        this.namaKerajinan = namaKerajinan;
    }

    public int getKodeKerajinan() {
        return kodeKerajinan;
    }

    public String getBahan() {
        return bahan;
    }

    public String getNamaKerajinan() {
        return namaKerajinan;
    }
}

File: cpp/2nd_Week/ContohKasusKayu/HasilKerajinan.cpp
#include <iostream>
#include <string>

using namespace std;

class HasilKerajinan
{
    private:
        /* data */
        int kodeKerajinan;
        string bahan, namaKerajinan;
    public:
        HasilKerajinan(){};

        void setKodeKerajinan(int kodeKerajinan) {
            this->kodeKerajinan = kodeKerajinan;
        }

        void setBahan(string bahan) {
            this->bahan = bahan;
        }

        void setNamaKerajinan(string namaKerajinan) {
            this->namaKerajinan = namaKerajinan;
        }

        int getKodeKerajinan() {
            return kodeKerajinan;
        }

        string getBahan() {
            return bahan;
        }

        string getNamaKerajinan() {
            return namaKerajinan;
        }

        ~HasilKerajinan(){};

};
